### PR TITLE
Add tap-pardot

### PR DESCRIPTION
**Pull request by przywartya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+3. Enter project name as integration name (ex. tap-google) + select location: `dreamdata.io/integrations/`
+
+# Running tap tool for new client
+
+When project for integration exists, but specific client dataset not, then the dataset is created automatically.
+
+# Attention!
+
+Did you just waste 2 hours running you `target-bigquery` pipeline, but didn't do the SYNC?
+I feel you pain. To do that you need to:
```
Comment (przywartya) [inline README.md]: @paddie

File: tap_tool/sync.py [hunk 2]
```diff
@@ -116,3 +120,7 @@ def sync(
             bigquery,
             catalog,
         )
+    elif integration == "pardot":
+        return pardot(
+            company, secret, "FULL_TABLE", output, discover, bigquery, catalog,
```
Comment (JingLin0) [inline tap_tool/sync.py]: Is there a way to sync `INCREMENTALLY`?

File: README.md (hunk 1 above)
Comment (paddie) [reply README.md]: Hahahaha

File: tap_tool/sync.py [hunk 3]
```diff
@@ -10,11 +10,8 @@
 from tap_tool.integrations.microsoft_dynamics import microsoft_dynamics
 from tap_tool.integrations.bing_ads import bing_ads
 from tap_tool.integrations.pipedrive import pipedrive
-<<<<<<< HEAD
 from tap_tool.integrations.salesforce import salesforce
-=======
 from tap_tool.integrations.pardot import pardot
->>>>>>> Add pardot tap definition
```
[the remark this answers is not in the document]
Comment (paddie) [reply tap_tool/sync.py]: the reason why we leave them out, is that people might waste time writing a CR comment about that being left in by mistake, and then you removing them in another commit.

Good commit citizen practice :)

File: tap_tool/sync.py (hunk 2 above)
Comment (paddie) [reply tap_tool/sync.py]: 👏

File: README.md (hunk 1 above)
Comment (JingLin0) [reply README.md]: Haha, I would say "It is your destiny"

Comment (paddie) [reply README.md]: "take up meditation"?

Comment (przywartya) [reply README.md]: "just leave it and go live in the mountains" 💃

File: tap_tool/sync.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (przywartya) [reply tap_tool/sync.py]: Good citizen indeed!

File: tap_tool/utils.py [hunk 4]
```diff
@@ -67,7 +67,7 @@ def execute(
 
     if tap == "tap-googlesearch":
         catalog_cmd = ""
-    elif tap == "tap-hubspot" or "tap-salesforce":
```
Comment (paddie) [inline tap_tool/utils.py]: HAHAHAHA, did not catch this thing. to make this less likely to happen again, we could change it to a list instead.

```python
elif tap in ["tap-hubspot", "tap-salesforce"]:
  ...
```

but it is fine for now :D

File: README.md [hunk 5]
```diff
@@ -18,7 +18,8 @@ For example:
 
 When project for integration exists, but specific client dataset not, then the dataset is created automatically.
 
-# Attention!
+# Attention! DON'T USE FULL_TABLE
 
 Did you just waste 2 hours running you `target-bigquery` pipeline, but didn't do the SYNC?
-I feel you pain. To do that you need to:
+I feel your pain. Next time just use replication method INCREMENTAL. It will create a
+special table in bigquery with SYNC information.
```
Comment (paddie) [inline README.md]: the full_table should also create a `sync` table?

Comment (przywartya) [reply README.md]: It does not do it. I don't know if it should, I'm not an expert in that 👍

**Pull request merged**